vioscsi/vioscsi.c

```diff
@@ -485,7 +485,16 @@ ENTER_FN();
 
     /* Set num_queues and seg_max to some sane values, to keep "Static Driver Verification" happy */
     adaptExt->scsi_config.num_queues = 1;
-    adaptExt->scsi_config.seg_max = MAX_PHYS_SEGMENTS + 1;
+    /* Happy? (Begin) */
+    /* Getting virtqueue_size is prioritized over your happiness. I don't care. */
+    adaptExt->max_physical_breaks = min(
+                                        max(SCSI_MINIMUM_PHYSICAL_BREAKS, adaptExt->scsi_config.seg_max),
```

Comment:
`Should I do `SCSI_MINIMUM_PHYSICAL_BREAKS + 1` instead? (And also the clamp in for the registry)

Reply:
For the record, apparently the default value of `ConfigInfo->NumberOfPhysicalBreaks` is 0x11 (i.e. `17` / `SCSI_MINIMUM_PHYSICAL_BREAKS + 1`)...